README.md

```diff
@@ -197,6 +197,30 @@ it('should call the API', async () => {
 });
 ```
 
+### Tracking unused/missing injectables
+
+By default `magnetic-di` does not complain if an injectable is not used or if a dependency has not being replaced. In large codebases however, that might led to issues with stale, unused injectables or with lack of knowledge in what could be replaced. To ease introspection, the library provides a `stats` API that returns both `unused` and `missing` injectables.
+
+- `stats.unused()` returns an array of entries `{ get(), error() }` for all injectables that have not been used since `stats.reset()` has been called
+- `stats.missing()` returns an array of entries `{ get(), error() }` for all dependencies that have not been replaced since `stats.reset()` has been called
+
+This is an example of stats guard implementation using the returned `error()` helper:
+
+```js
+import { stats } from 'react-magnetic-di';
+
+beforeEach(() => {
+  // it's important to reset the stats after each test
+  stats.reset();
+});
+afterEach(() => {
+  stats.unused().forEach((entry) => {
+    // throw an error pointing at the test with the unused injectable
+    throw entry.error();
+  });
+});
+```
```

Comment:
👍 , and it's probably better to document `missing` a little later as it might require some good usecases and patterns.
A couple of years ago [I've called](https://github.com/theKashey/rewiremock/blob/master/README.md#isolation) `unused` as `reverse isolation`, so `missing` can be called __isolation__.

My idea was to "list" all deps which are ok not be used (`useState`,`useReducer`, many others) to catch new dependencies and keep SUT "boxed" in well-known mocks. 
Probably it's better to do the opposite and list some deps which are more likely to be mocked, and create flexibility for any other.
Both versions can be implemented on top of a given API. But... let's first clearly document use cases and then document this feature. It just might need some time, while the rest is "ready"